index.html

```diff
@@ -4983,6 +4983,93 @@ <h3>Base Context</h3>
           </p>
       </section>
 
+      <section class="normative">
+        <h3>Vocabularies</h3>
+
+        <p>
+Implementations MUST ensure that the following vocabulary URLs used in the base
+context ultimately resolve to the following files, which are normative:
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>URL</th>
+              <th>Media Type</th>
+              <th>Content</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+https://www.w3.org/2018/credentials#
+              </td>
+              <td>
+text/html
+              </td>
+              <td>
+https://www.w3.org/2018/credentials/index.html
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://www.w3.org/2018/credentials#
+              </td>
+              <td>
+application/ld+json
+              </td>
+              <td>
+https://www.w3.org/2018/credentials/index.jsonld
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://schema.org/
+              </td>
+              <td>
+text/html
+              </td>
+              <td>
+https://schema.org/
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://schema.org/
+              </td>
+              <td>
+application/ld+json
+              </td>
+              <td>
+https://schema.org/version/latest/schemaorg-current-https.jsonld
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://w3id.org/security#
+              </td>
+              <td>
+text/html
+              </td>
+              <td>
+https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.html
```

Comment:
This should be to a w3c origin, not github.

Reply:
I agree, but the appropriate URL doesn't exist at this time.

I would expect that this URL will end up being `https://www.w3.org/ns/security`, but the VCWG hasn't set up that URL yet. I'll add an issue marker to that effect to the PR.